index.js

```diff
@@ -373,62 +373,86 @@
   ];
 
   //  ValidDate :: Type
-  $.ValidDate = $.NullaryType(
+  $.ValidDate = NullaryType(
     'sanctuary-def/ValidDate',
     function(x) { return $.Date.test(x) && !isNaN(x.valueOf()); }
   );
 
+  //  PositiveNumber :: Type
+  $.PositiveNumber = NullaryType(
+    'sanctuary-def/PositiveNumber',
+    function(x) { return $.Number.test(x) && x > 0; }
```

Comment:
You declare `ValidNumber` below as `var ValidNumber = $ValidNumber =` so that you may use `ValidNumber` for other type definitions elsewhere in this scope. Here though you refer to `$.Number` because the way the built in types are declared. Why the inconsistency? And, for that matter, why are these types not included in `$.env`?